src/pages/wallets/Wallet.js

```diff
@@ -112,15 +121,60 @@ function Wallet(props) {
       dataField: "collateral",
       text: "collateral",
       sort: true,
-      formatter: (cell, row) => <Value value={cell} decimals={2} compact />,
+      formatExtraData: { timePeriod },
+      formatter: (cell, row) => (
+        <>
```

Comment:
Since you have a wrapping div, you don't need the empty `<></>` around everything as the div will do the same job